pull_twitter.py

```diff
@@ -46,5 +65,28 @@
     # tweepy client
     client = Client(bearer_token=os.environ['TW_BEARER_TOKEN'], wait_on_rate_limit=True)
 
+    # Create output directories
+    dt_fmt = '%Y-%m-%d %H:%M:%S' if os.name == 'posix' else '%Y-%m-%d %H.%M.%S'
```

Comment:
This is cool, but I think it would be nicer to just use a single format that works for both Windows and *nix systems.

Reply:
Format changed to simply use periods instead of colons for all file systems.